### PR TITLE
tag: Adding {{Cleanup colors}} + {{Overcoloured}}

### LINKED ISSUE #1847: tag: add {{Cleanup colors}} and {{Overcoloured}}
Requested by Freedom4U at https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Addition_of_cleanup_template

### PR DESCRIPTION
**Pull request by lomrjyo**
will close #1847

### CODE REVIEW

**Comment by NovemLinguae** [comment]
@lomrjyo. It looks like you created this PR on your fork's master branch instead of on its own branch. So I went to go edit it to make some changes but it didn't work. In the future, you can avoid this problem by creating your PRs on their own branches instead of the master branch.

So I can't make the changes for you. I'll leave them as code review comments instead. If you make the changes, I'll merge this.

Thanks for the patch.

File: modules/friendlytag.js [hunk 1]
```diff
@@ -903,6 +903,10 @@ Twinkle.tag.article.tagList = {
 		'Categories': [
 			{ tag: 'Improve categories', description: 'needs additional or more specific categories', excludeMI: true },
 			{ tag: 'Uncategorized', description: 'not added to any categories', excludeMI: true }
+		],
+		'Colors': [
```
Comment (NovemLinguae) [inline modules/friendlytag.js:907]: Rename to 'Accessibility'

File: modules/friendlytag.js [hunk 2]
```diff
@@ -903,6 +903,10 @@ Twinkle.tag.article.tagList = {
 		'Categories': [
```
Comment (NovemLinguae) [inline modules/friendlytag.js:903]: Cut and paste both 'Categories' and 'Accessibility' so that they are in alphabetical order

**Review by NovemLinguae — CHANGES_REQUESTED** [review]

File: modules/friendlytag.js (hunk 2 above)
Comment (DreamRimmer) [reply modules/friendlytag.js:903]: @NovemLinguae, they have not been active for a year, so can I create a new PR?

Comment (NovemLinguae) [reply modules/friendlytag.js:903]: Sure go ahead. If you can add a comment in your new PR reminding me to close this one, that'd be appreciated.

**Comment by DreamRimmer** [comment]
@NovemLinguae, new PR is #2026

**Pull request closed without merging**